.circleci/config.yml

```diff
@@ -72,6 +73,9 @@ workflows:
           context:
             - filecoin-dockerhub-publish
           filters:
+            # Do not run on branch builds
+            branches:
+              ignore: /.*/
```

Comment:
Looks like this job was running for every branch which I believe was not intended @coryschwartz ?

The change here will only run the job on semver tagged commits.